historic_cadastre/__init__.py

```diff
@@ -10,6 +14,39 @@
 import yaml
 
 from historic_cadastre.lib import dbreflection
+from historic_cadastre.lib.authentication import RemoteUserAuthenticationPolicy
+
+
+def _create_get_user_from_request(settings):
+    def get_user_from_request(request):
+        """ Return the User object for the request.
+        Return ``None`` if:
+        * user is anonymous
+        * it does not exist in the database
+        * the referer is invalid
+        """
+        from historic_cadastre.models import DBSession, AuthenticationUser
+
+        if not hasattr(request, "_user"):
```

Comment:
Nothing, because `request.user` will be `None` and if this is the case and the user wants to get to a protected place, it will only return a HTTPForbidden code.